### ISSUE TITLE
EDSR Model PSNR not improving

### ISSUE THREAD

**Issue by purijs** [krasserm/super-resolution#69]
I'm trying to train a model on custom data, however the PSNR is stuck around `19` while loss is constantly going down, what can I infer from this?

In what sense is the model improving or not improving? My data is around 300 images, scaled at 4x

A snap: `100000/400000: loss = 17.654, PSNR = 19.784256 (611.89s)`

**Comment by tuvovan** [comment]
any updates?
